src/test/java/unbanner/StudentControllerTest.java

```diff
@@ -2,12 +2,14 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-
 import org.junit.Test;
+import org.junit.Assert.*;
```

Comment:
Can you unglob this? Style check doesn't like '.*'

Also, this should be above org.junit.Test

Reply:
import static org.junit.Assert.assertEquals; 
is this okay with Style Check?

Reply:
Yep, that should work. Style Checker just wants imports to be explicit and in alphabetical order.